upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+    ALTER TABLE ta_tags CHANGE name name VARCHAR(191) NOT NULL;
+    ALTER TABLE we_domains CHANGE url url VARCHAR(191) NOT NULL;
+    ALTER TABLE we_analytics CHANGE webspace_key webspace_key VARCHAR(191) NOT NULL;
+    ALTER TABLE ro_routes CHANGE path path VARCHAR(191) NOT NULL, CHANGE entity_class entity_class VARCHAR(191) NOT NULL, CHANGE entity_id entity_id VARCHAR(191) NOT NULL;
+    ALTER TABLE me_collection_meta CHANGE title title VARCHAR(191) NOT NULL;
+    ALTER TABLE me_file_version_meta CHANGE title title VARCHAR(191) NOT NULL;
+    ALTER TABLE me_file_versions CHANGE name name VARCHAR(191) NOT NULL;
+
+If you are using jackalope with doctrine-dbal instead of jackrabbit you also have to execute the following queries:
+
+.. code-block:: sql
+
+    ALTER TABLE phpcr_binarydata CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_internal_index_types CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_namespaces CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_nodes CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_nodes_references CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_nodes_weakreferences CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_type_childs CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_type_nodes CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_type_props CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_workspaces CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE phpcr_namespaces CHANGE prefix prefix VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_workspaces CHANGE name name VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_nodes CHANGE path path VARCHAR(191) NOT NULL, CHANGE parent parent VARCHAR(191) NOT NULL, CHANGE local_name local_name VARCHAR(191) NOT NULL, CHANGE namespace namespace VARCHAR(191) NOT NULL, CHANGE workspace_name workspace_name VARCHAR(191) NOT NULL, CHANGE identifier identifier VARCHAR(191) NOT NULL, CHANGE type type VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_internal_index_types CHANGE type type VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_binarydata CHANGE property_name property_name VARCHAR(191) NOT NULL, CHANGE workspace_name workspace_name VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_nodes_references CHANGE source_property_name source_property_name VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_nodes_weakreferences CHANGE source_property_name source_property_name VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_type_nodes CHANGE name name VARCHAR(191) NOT NULL;
+    ALTER TABLE phpcr_type_props CHANGE name name VARCHAR(191) NOT NULL;
+
+**5. Execute the following SQL statements to migrate your data**
+
+.. code-block:: sql
```

Comment:
maybe add a short description that these statements are needed for the sulu-internal database structure changes